addons/gamepad/addon.json

```diff
@@ -22,7 +22,7 @@
       "matches": ["https://scratch.mit.edu/projects/*"]
     }
   ],
-  "tags": [],
+  "tags": ["beta"],
```

Comment:
It's definitely an editor addon?

Reply:
I'm not sure. This addon is essential JoyToKey as an addon. You could use it in the editor, but I suspect more people will use it outside of the editor

Reply:
60fps, mute project, and etc are often used outside the editor, yet they have `editor` tags.
Plus, it needs either `community` or `editor` because of how the settings page is made, and `editor` is definitely more applicable here than `community`.